### PR TITLE
Added 3 New Kudos!

### PR DESCRIPTION
**Pull request by yash412**
Added three new kudos! :') 

- Natural Coder 
eth8

- Bug Detective 
eth9

- Appreciation
eth10

### CODE REVIEW

File: kudos.yaml [hunk 1]
```diff
@@ -2314,3 +2314,38 @@
...
+  priceFinney: 40
+  numClonesAllowed: 50
+  tags:
+    - coder
+    - bug
+    - debug
+    - debugger 
+  image: bug21.svg
+    - name: Natural_Coder
+  to_address: 0xD27D4fE82aCF5eaBb78723eC0F624c8127c64972
```
Comment (owocki) [inline kudos.yaml]: mind putting the address in single quotes pls?

**Comment by owocki** [comment]
looks good! just one small nitpick on the yaml file.

**Comment by yash412** [comment]
Alright just a second 👍

**Comment by yash412** [comment]
@owocki , done. I did notice every other address is also not in quotes? Need me to change that?

**Comment by yash412** [comment]
Converted all the addresses! @owocki

**Comment by owocki** [comment]
Thanks. @yash412 you are my favorite :)

**Comment by owocki** [comment]
live on https://gitcoin.co/kudos/marketplace/

**Pull request merged**